### DISCUSSION TITLE
请问有没有桌面滚动歌词的功能和评论功能？

### DISCUSSION THREAD

**Discussion by zhouchangming0727** [qier222/YesPlayMusic#1255] [Q&A]
# 尽量每个 issue 只提一个 bug 或新功能

### 提新 issue 前请确认 👉

- 没人提过这个 issue（[这里看所有 issue](https://github.com/qier222/YesPlayMusic/issues)）
- 项目的 Todo 里没有与你 issue 相关的内容（[这里看 Todo](https://github.com/qier222/YesPlayMusic/projects/1)）

### 反馈 bug 需要的信息

- 用的是网页版还是客户端
- 浏览器名称或电脑操作系统
- 控制台 Console 页面的截图（按 F12 可打开控制台）

**Comment by memorydream** [comment]
无任何社交功能是yesplaymusic的卖点之一
没有桌面歌词，可能以后也不会打算加入，如果你需要，可以使用这个fork中的版本
https://github.com/shih-liang/YesPlayMusicOSD
